include/Dialects/LinalgExt/LinalgExtOps.td

```diff
@@ -191,4 +191,129 @@ def LinalgExt_TileYieldOp : LinalgExt_PureOp<"tile_yield", [
   let assemblyFormat = "attr-dict ($operands^ `:` type($operands))?";
 }
 
+def LinalgExt_InParallelOp : LinalgExt_PureOp<"in_parallel", [
```

Comment:
Maybe add some notes that this is very similar to an scf.for_op but:
- without iter_args 
- a special terminator (that would really like to be a dependent-attribute that can refer to SSA values)
- with a single thread id in the range 0:num_threads:1 instead of an induction variable.

Reply:
Done.